content/hardware/04.pro/boards/portenta-x8/tutorials/10.datalogging-iot/content.md

```diff
@@ -75,14 +75,26 @@ Let's start by configuring the MQTT broker!
 
 ## Installing Mosquitto
 
-Let's start by creating a new directory in our Portenta X8 called `mqtt`; inside this directory, we are going to make a file named `docker-compose.yml`:
+Let's start by creating a new directory in our Portenta X8 `home` directory called `mqtt`; inside this directory, we are going to make a file named `docker-compose.yml` using the following commands:
 
 ```
-$ mkdir mqtt
-$ cd mqtt
-$ export TERM=xterm
-$ stty rows 36 cols 150
-$ sudo vi docker-compose.yml
+mkdir mqtt
+```
+
+```
+cd mqtt
+```
+
+```
+export TERM=xterm
+```
+
+```
+stty rows 36 cols 150
+```
+
+```
+sudo vi docker-compose.yml

```

Comment:
I love that we are taking into account the problem that we mentioned in some meetings.

Not 100% convinced about the visual result:
![image](https://user-images.githubusercontent.com/7687624/225892481-a63bce78-7996-430e-8055-cfe79494cec9.png)

Ideally that would be solved by having some small lines explaining each command in case if necessary but we don't need that now and at the same time is like: "create a folder", go to the folder, etc, not sure.

Let's go with this change, thanks!

Reply:
Yes, Markdown is not the best for console; I am going to keep this in mind for future work.